mockery.cabal

```diff
@@ -1,9 +1,9 @@
--- This file has been generated from package.yaml by hpack version 0.5.4.
+-- This file has been generated from package.yaml by hpack version 0.9.0.
 --
 -- see: https://github.com/sol/hpack
 
 name:           mockery
-version:        0.3.2
+version:        0.3.1
```

Comment:
I thought about omitting this line, but I think it is useful to point out that the version in the cabal file did not match the version in the package.yaml, which suggests that the cabal file was modified directly.